src/core/parcel-exceptions-data.ts

```diff
@@ -0,0 +1,330 @@
+/**
+ * Data file containing parcel exception coordinates
+ * @internal
+ */
+
+/**
+ * Block 1: Coordinates X=151-163, Y=59-143
+ * Parcels on the eastern border where X exceeds the standard maximum of 150
+ */
+export const block_1 = [
+  '151,59',
```

Comment:
Can these block definitions be replaced so the parcels are checked logically?
For example, this could be computed as x >= 151 <= 152 should have y >= 60 <= 143 for it to be a correct parcel position.

Reply:
Review the refactor please